docs/superpowers/plans/2026-08-14-changesets-publish.md

```
@@ -0,0 +1,466 @@
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "baseBranch": "main",
+  "updateInternalDependencies": "patch",
+  "ignore": []
+}
+```
+
+> 说明：新版 `@changesets/config` schema 仅接受 `"minor" | "patch"`，不再接受 `false`。单包仓库无 internal 依赖，此字段无实际作用，用 `"patch"` 即可。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)